.azure-pipelines/client.yml

```diff
@@ -87,6 +87,18 @@ jobs:
       vmImage: '$(OSVmImage)'
 
     steps:
+      - powershell: |
+          Invoke-WebRequest -Uri "https://github.com/chidozieononiwu/azure-sdk-for-net/releases/download/verifyAgentOS_v1.1/verify_agent_os.zip" `
```

Comment:
Yes this is along the lines of what I was looking for. I'd like to get this pulling from the azure-sdk-tools release before we try to merge this however. I'm fine if you want to manually create a release there to bootstrap but ideally you would go ahead and make the necessary changes to azure-sdk-tools build pipeline to produce the release.

Reply:
Ok, I will see if i can get the changes setup in the azure-sdk-tools today otherwise I will create it manually so that this PR can be merged. I think I also need to set this up for all the repos?

Reply:
Yes you will need to set it up for all repos which is why I'd like the url to closely match what we expect it to look like from the azure-sdk-tools build before updating all the repos.